sky/spot/spot_utils.py

```diff
@@ -42,6 +42,7 @@
                                '{status_str}.[/] It may take a few minutes.')
 _JOB_CANCELLED_MESSAGE = ('[bold cyan]Waiting for the job status to be updated.'
                           '[/] It may take a minute.')
+_FINAL_SPOT_STATUS_WAIT_TIMEOUT_SECONDS = 10
```

Comment:
Comment on what this does? Not clear to me from L324-L325.

Reply:
Added. How does that look?
```
# The maximum time to wait for the spot job status to transition to terminal
# state, after the job finished. This is a safeguard to avoid the case where
# the spot job status fails to be updated and keep the `sky spot logs` blocking
# for a long time.
```